.github/workflows/tx-pull.yml

```diff
@@ -0,0 +1,91 @@
+on:
+  push:
+    branches:
+    - main
+
+jobs:
+  install-tx-client-and-pull-translations:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Cache public folder
+      uses: actions/cache@v3
+      with:
+        path: public
+        key: ${{ github.sha }}
+    - name: Install Transifex client
+      run: |
+        curl -o- https://raw.githubusercontent.com/transifex/cli/master/install.sh | bash
+    - name: Pull translations from Transifex
+      run: |
+        tx -t ${{ secrets.TX_TOKEN }} pull -a
+
+  pull-request:
```

Comment:
Do we need two jobs here? iiuc everything could be done in one, removing need for passing things in cache

Reply:
ack, let's leave it as-is, we can simplify in future PR.